bin/yb-ctl

```diff
@@ -1342,6 +1355,18 @@ class ClusterControl:
         alive (based on valid PID), then this function returns True. Anything else will end up
         returning False.
         """
+        # Wait for master and tserver processes to be ready.
+        def call_check_for_processes(should_get_error):
+            pids = self.for_all_daemons(self.get_pid)
+            if self.args.verbose:
+                logging.info("Daemon PIDs found after starting daemons: {}".format(pids))
+            return all(sum(pids.values(), []))
+
+        logging.info("Waiting for master and tserver processes to be up.")
+        if not retry_call_with_timeout(call_check_for_processes, self.options.timeout):
```

Comment:
instead of the overall timeout here, I'd add a new constant at the top, maybe like 5 seconds, since really, if the processes do not come up in a couple of seconds, they likely crashed

Reply:
Forgot to mention that I didn't work on this.  So this timeout is hard-coded and not exposed to the user?